scripts/navFunc.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+console.log('hello');
+// $('a').on('click', function(event){
```

Comment:
Please take out commented code. I'l start docking points starting next week for this.